identity/src/main/java/com/android/identity/DataTransportBle.java

```diff
@@ -52,6 +52,8 @@ public abstract class DataTransportBle extends DataTransport {
     private static final String TAG = "DataTransportBle";
     @LoggingFlag
     protected int mLoggingFlags;
+    // Indicates to use L2CAP for BLE transfer by default
+    protected boolean mSupportL2CAP = true;
 
```

Comment:
I would call this `mUseL2CAPIfAvailable` and change the comment to "Indicates whether L2CAP should be used if it's available".

Reply:
Done